apps/dashboard/src/modules/admin/views/AdminUserOverView.vue

```diff
@@ -190,20 +190,36 @@ function debounce(func: (skip: number) => Promise<void>, delay: number): (skip:
 }
 
 const apiCall: (skip: number) => Promise<void> = async (skip: number) => {
-  await apiService.user
-    .getAllUsers(
-      Number.MAX_SAFE_INTEGER,
-      skip,
-      searchQuery.value.split(' ')[0] || '',
-      isActiveFilter.value,
-      ofAgeFilter.value,
-      undefined,
-      filters.value.type.value || undefined
-    )
-    .then((response) => {
-      totalRecords.value = response.data._pagination.count || 0;
-      allUsers.value = response.data.records;
-    });
+    await apiService.user
+        .getAllUsers(
+            DEFAULT_PAGINATION_MAX,
+            skip,
+            searchQuery.value.split(' ')[0] || '',
+            isActiveFilter.value,
+            ofAgeFilter.value,
+            undefined,
+            filters.value.type.value || undefined
+        )
+        .then((response) => {
+          totalRecords.value = response.data._pagination.count || 0;
+          allUsers.value = response.data.records;
+        });
+    if ( !isActiveFilter.value ){
+      await apiService.user
+          .getAllUsers(
+              DEFAULT_PAGINATION_MAX,
+              skip,
+              searchQuery.value.split(' ')[0] || '',
+              !isActiveFilter.value,
+              ofAgeFilter.value,
+              undefined,
+              filters.value.type.value || undefined
+          )
+          .then((response) => {
+            totalRecords.value += (response.data._pagination.count || 0);
+            allUsers.value = allUsers.value.concat(response.data.records);
+          });
+    }
```

Comment:
What does this achieve? why the change?

Reply:
This was for another issue, I did an oepsie